index.js

```diff
@@ -124,6 +125,31 @@ module.exports = class Kernel {
     }
   }
 
+  createHandler (create) {
+    let code = create.data
+
+    // Inject metering
+    if (Utils.isWASMCode(code)) {
+      code = this.callHandler({ to: meteringContract, data: code }).returnValue
+    }
+
+    let address = Utils.newAccountAddress(create.from, code)
+
+    this.environment.addAccount(address.toString(), {
+      balance: create.value,
+      code: code
+    })
+
+    // Run code and take return value as contract code
+    code = this.callHandler({ from: create.from, to: address, gasLimit: create.gasLimit }).returnValue
+
+    this.environment.state.get(address.toString()).set('code', code)
+
+    return {
+      accountCreated: address
```

Comment:
No, it should return execution status / gas used. Basically it is a special case of call if we follow what other VMs do.